Documentation alert

braswelljr opened the pull request:
## Description
Alert for New Documentation at [https://glab-cli.io](https://glab-cli.io)

## Related Issue
<!--- This project only accepts pull requests related to open issues -->
<!--- If suggesting a new feature or change, please discuss it in an issue first -->
<!--- If fixing a bug, there should be an issue describing it with steps to reproduce -->
Resolves #[issue_number]

## How Has This Been Tested?
<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other are
as of the code, etc. -->

## Screenshots (if appropriate)
![Screenshot from 2021-11-02 11-38-54](https://user-images.githubusercontent.com/53278231/139839584-b0cd2fb5-4666-4af0-9c85-79f9b5ecce00.png)
![Screenshot from 2021-11-02 11-40-46](https://user-images.githubusercontent.com/53278231/139839826-9f097272-06fd-4ba7-bf35-025717861516.png)

## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [X] Documentation
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (Related to CI or Packaging to platforms)

profclems committed:
docs: markdown generation

profclems committed:
feat: switch to cobra\'s v2 bash completion

This adds description of commands to bash completion script.
Also introduces `--no-desc` flag to allow users turn off description.

profclems committed:
Merge pull request #796 from profclems/bash-v2

feat: switch to cobra's v2 bash completion

profclems committed:
{go.mod, go.sum}: update dependencies

profclems committed:
--amend

profclems committed:
Merge pull request #797 from profclems/bump-deps

{go.mod, go.sum}: update dependencies

profclems committed:
feat: `variable list` command

marckhouzam committed:
Apply --no-desc to every shell for completion

This change also enables the use of descriptions for Powershell.

Signed-off-by: Marc Khouzam <marc.khouzam@gmail.com>

profclems committed:
Merge pull request #801 from marckhouzam/feat/compNoDesc

profclems committed:
Update README.md

profclems committed:
Merge pull request #807 from profclems/profclems-patch-1

Update README.md

profclems committed:
commands/ssh-key: add commands for managing ssh keys

This change add `ssh-key` commands with the following subcommands:
- get
- list
- add

profclems committed:
Merge pull request #808 from profclems/feat-ssh-key

commands/ssh-key: add commands for managing ssh keys

profclems committed:
commands: improve scriptability

Resolves #809

profclems committed:
Merge pull request #799 from profclems/ft-var-list

feat: `variable list` command

profclems committed:
Merge pull request #811 from profclems/scriptability

commands: improve scriptability

Commit:
docs(update): v1.19.0

profclems committed:
internal/config: migrate local config to `.git/glab-cli`

To set a local config, glab writes to .glab-cli/config/config.yaml and
to hide it, writes to .gitignore.
This change migrates the local config to .git/glab-cli to avoid messing with the .gitignore of the current project.

profclems committed:
Merge pull request #813 from profclems/config-fix

internal/config: migrate local config to `.git/glab-cli`

profclems committed:
goreleaser: fix nfpms install dir to use /usr/bin

Closes #816

profclems committed:
Merge pull request #817 from profclems/goreleaser-fix

goreleaser: fix nfpms install dir to use /usr/bin

guyfedwards committed:
feat: fill MR desc with all commit bodies

adds --fill-commit-body flag to add all the commit bodies to the MR description
when creating an my with --fill

closes #795

Bostigger committed:
commands/ssh-key: fix wrong usage description in `ssh-key add`

profclems committed:
Merge pull request #820 from Bostigger/trunk

commands/ssh-key: fix wrong usage description in `ssh-key add`

clwluvw committed:
fix: allow token or stdin in non interactive shells

Signed-off-by: Seena Fallah <seenafallah@gmail.com>

zemzale committed:
feat(cmdutils): Add group milestones to the prompt

When creating issues we prompted only project milestones. This would
break workflows where group milestones were used.

They are two different APIs so we have to normalize their output and
merge it to one.

Issue #698

zemzale committed:
Merge pull request #823 from clwluvw/noninteractive-login

fix: allow token or stdin in non interactive shells

profclems committed:
move contributing guide project root

profclems committed:
Rename .github/CODE_OF_CONDUCT.md to CODE_OF_CONDUCT.md

profclems committed:
Update README.md

profclems committed:
Update CONTRIBUTING.md

profclems committed:
Merge pull request #819 from guyfedwards/add-fill-commit-body

feat: fill MR desc with all commit bodies

rdegbe committed:
Update ssh-key.go

TokyoQ committed:
fix(pipeline): allow pipeline parameters to use colons in the values

    Fixes #829

zemzale committed:
Merge pull request #830 from TokyoQ/fix/colon_pipeline_parameters

fix(pipeline): allow pipeline parameters to use colons in the values

zemzale committed:
Merge pull request #827 from rdegbe/trunk

Add `--repo` flag to ssh-key subcommands

rciorba committed:
Handle paginated response in pipeline.GetPipelineJobs

The `ci trace` and `ci status` commands would previously ignore pagination and
only show the first jobs, if the job number is large.

This MR changes pipeline.GetPipelineJobs to handle paginated responses, and also
increases the PerPage to a larger value (100) to minimize the number of requests
needed.

Fixes #779

profclems committed:
Merge pull request #832 from rciorba/779-fix-truncated-jobs-in-ci-status-and-ci-trace

Handle paginated response in pipeline.GetPipelineJobs

profclems committed:
Merge pull request #824 from zemzale/feat-all-milestones

feat(cmdutils): Add project and group milestone prompt

profclems committed:
fix: error 404 when creating release

Resolves #835

profclems committed:
Merge pull request #836 from profclems/release-fix

fix: error 404 when creating release

profclems committed:
api: fix wrong comment

profclems committed:
{go.mod,go.sum}: update dependencies

profclems committed:
Merge pull request #837 from profclems/deps-update

{go.mod,go.sum}: update dependencies

addshore committed:
Move glinstance to pkg

addshore committed:
Login using default hostname string

addshore committed:
auth login use OverridableDefault

profclems committed:
Merge pull request #845 from addshore/mwcli-modifications

feat: allow reuse of glab in other CLI contexts

profclems committed:
fix debian packaging installer location

Closes #847 and #846

Commit:
docs(update): v1.21.1

drewrisinger committed:
docs: remove --autofill references

It appears the --autofill option to ``glab mr create`` was removed, but references remained.

hoelzro committed:
fix(ci trace): exit normally when prompts are interrupted

…specifically, interrupted via SIGINT (CTRL-C).  Otherwise,
the first job is chosen, which isn't a huge deal, but isn't really
consistent with the user's wish to exit the program early

d3adb5 committed:
add remoteName flag to 'create project' command

By default glab would add the created project's repository URL to the a
remote named 'origin', which fails when there already is such a remote.

This commit adds the option to specify a remote name for glab to add in
the local git repository, so that the user may specify a different one,
should 'origin' be already taken.

Closes #803.

szemek committed:
docs(readme): update Fosshost.org logo location

- in README.md replace FosshostLogo.png with fosshost-logo.png as previous location no longer exists (return 404)

profclems committed:
Merge pull request #849 from hoelzro/trunk

fix(ci trace): exit normally when prompts are interrupted

profclems committed:
Merge pull request #851 from szemek/fosshost-logo

docs(readme): update Fosshost.org logo location

profclems committed:
Merge pull request #848 from drewrisinger/dr-pr-remove-autofill-docs

docs: remove --autofill references

profclems committed:
Merge pull request #850 from skewerr/add-create-remote-name-flag

Add remoteName flag to glab repo create

hoelzro committed:
chore: use errors.Is instead of error equality

This might be needlessly pedantic, but it insulates against potential
future changes in error wrapping

pataar committed:
feat(mr create): add --reviewer parameter

pataar committed:
docs(mr create): add --reviewer parameter to docs

pataar committed:
docs(mr create): remove doc entry as it's automatically generated

pataar committed:
feat(mr create): add --squash parameter

pataar committed:
feat(mr update): add --squash parameter

pataar committed:
docs(template): use different header styles for the Pull Request template

pataar committed:
chore: remove colon

profclems committed:
Merge pull request #857 from pataar/patch-5

docs(template): use different header styles for the PR template

zemzale committed:
Merge pull request #853 from hoelzro/errors-is

chore: use errors.Is instead of error equality

zemzale committed:
Merge pull request #854 from pataar/trunk

feat(mr create): add --reviewer parameter

pataar committed:
refactor: rename Squash to SquashBeforeMerge

pataar committed:
refactor: rename Squash to SquashBeforeMerge

profclems committed:
Merge pull request #855 from pataar/patch-3

feat(mr create): add `--squash-before-merge` parameter

pataar committed:
ci: fix the linter by targeting the pull request instead of the target

Commit:
fix ci pipeline view incorrect timestamp being used

Fixes #858

profclems committed:
Merge pull request #861 from pataar/feature/fix_ci_lint

ci: fix the linter by targeting the pr instead of the pr target

profclems committed:
Merge pull request #864 from junderhill/pipeline-view-timestamp

fix ci pipeline view incorrect timestamp being used

b4nst committed:
fix: weird condition when prompt disabled

TheArchDev committed:
docs(README): add awesome-go badge to README

glab is now listed in awesome-go repo so follow guidance to add badge
https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md

TheArchDev committed:
Update README.md

Co-authored-by: Clement Sam <clementsam75@gmail.com>

TheArchDev committed:
Update README.md

profclems committed:
Merge pull request #867 from TheArchDev/awesome-badge

docs(README): add awesome-go badge to README

szemek committed:
feat(command/ci/retry): Add new command to retry CI jobs

szemek committed:
Apply improvements from code review

szemek committed:
Handle invalid job id

profclems committed:
Merge pull request #868 from szemek/ci-retry

feat(command/ci/retry): Add new command to retry CI jobs

EduardBaer committed:
fix(ci status): add check for pending pipelines

hoelzro committed:
fix(mr-create): escape titles in new MR preview URLs

Fixes GH #833

profclems committed:
Merge pull request #869 from EduardBaer/pipeline-pending

fix(ci status): add check for pending pipelines

profclems committed:
Merge pull request #870 from hoelzro/mr-create-escape-spaces

Escape titles in new MR preview URLs

profclems committed:
Merge pull request #856 from pataar/patch-4

feat(mr update): add `--squash-before-merge` parameter

Commit:
fix: obsolete % char for weights

This change removes the additional %-char that shows up when handing
over a new issue to the gitlab GUI.

fixes #874

profclems committed:
Merge pull request #876 from Spreadcat/weight_issue

fix: obsolete % char for weights

bradym committed:
fix: allow setting variable without using --value (#881)

profclems committed:
update dependencies (#884)

profclems committed:
commands/project/create: fix typo

Commit:
Update docs for `ci run` to include passing variables

hoelzro committed:
Add support for terminal hyperlinks (#852)

* feat: Add ground work for supporting hyperlinks to iostreams

See GH #812

See https://gist.github.com/egmontkob/eb114294efbcd5adb1944c9f3cb5feda
for technical details behind the hyperlink escape sequence

For now hyperlinks are enabled as long as long as standard output is a
TTY - we'll want to add some additional detection/flags down the road

* feat: use terminal hyperlinks when listing issues/MRs

See GH #812

* feat(hyperlinks): Add display_hyperlinks config option

* chore(hyperlinks): Clean up Hyperlink API a bit

* feat(hyperlinks): default hyperlinks to off

I think hyperlink display should default to off while the feature is new
so that users can play with it

* feat(hyperlinks): Implement FORCE_HYPERLINKS

Setting the FORCE_HYPERLINKS environment variable forces glab to render
hyperlinks when it can - otherwise, it falls back to display_hyperlinks
from the config, which tells glab whether or not to enable hyperlinks
based on whether or not we're outputting to a TTY.

* fix(hyperlink): accommodate hyperlink OSC in string stripping

* fix: Fix incorrect web_url in fixture data

* test(hyperlink): Add tests for hyperlinks in issues list

* test(hyperlink): Add tests for truncating text with hyperlink OSC

* fix(hyperlinks): accommodate hyperlinks in Truncate

* test(hyperlinks): Add tests for showing hyperlinks in mr list

* docs(hyperlink): Add docs for FORCE_HYPERLINKS and display_hyperlinks

* fix(hyperlinks): simplify truncate algorithm

…while also fixing tableprinter tests, which expect that the first
grapheme provided to Truncate is always preserved

* fix(hyperlinks): Deactivate hyperlinks in auto mode if we're writing to a pager

zemzale committed:
Merge pull request #885 from junderhill/trunk

Update docs for `glab pipeline run` to include variables parameter

zemzale committed:
Merge pull request #865 from b4nst/patch-1

fix: weird condition when prompt disabled

braswelljr committed:
New documentation alert

braswelljr committed:
Get Documentation up to date

profclems commented on README.md:
```diff
@@ -98,9 +98,11 @@ Table of Contents
 
 ## Installation
-Download a binary suitable for your OS at the [releases page](https://github.com/profclems/glab/releases/latest).
+Download a binary suitable for your OS at the [releases page](https://github.com/profclems/glab/releases/latest)
```
We want to maintain the period at the end.

profclems commented on README.md:
```diff
@@ -98,9 +98,11 @@ Table of Contents
 
 ## Learn More
 Read the [documentation](https://glab.readthedocs.io/) for more information on this tool.
+### Documentation
+Get the new documentation on [`https://glab-cli.io`](https://glab-cli.io) and contribute to the new documentation on [https://github.com/braswelljr/glab-docs](https://github.com/braswelljr/glab-docs).
```
Remove the documentation header. 

I'm thinking maybe we could move the docs from the https://github.com/braswelljr/glab-docs repo to the /docs folder in this repo.

profclems reviewed (changes requested):
Great work @braswelljr! 

Thanks so much for working on this. We may have to decide whether to move the docs from the https://github.com/braswelljr/glab-docs repo to the /docs folder in this repo or maintain it at the current repo.

bradym committed:
feature: add variable delete command (#890)

* Move IsValidKey to new variableutils package

* add variable delete command

* remove unnecessary alias

* wip: add tests

* working tests

profclems committed:
Require Go 1.17+ (#898)

profclems committed:
mr: deprecate `mr for` command

The `mr for` command works just like create but accepts
issue ID as argument.

It creates a related merge request for the specified.
But for some reasons, it lacks some options available
when using `mr create`.

This change deprecates the `mr for` command and
adds additional options to `mr create` to perform
be able to create related merge request for issues

zemzale committed:
Merge pull request #900 from profclems/deprecate-mr-for

mr: deprecate `mr for` command

braswelljr commented:
I am glad my work has been approved.

I am ready to do what is best and help improve and make sure we deliver a
good and user friendly service to the open source community.

On Thu, 4 Nov 2021, 2:33 pm codecov[bot], ***@***.***> wrote:

> Codecov
> <https://codecov.io/gh/profclems/glab/pull/894?src=pr&el=h1&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Clement+Sam>
> Report
>
> Merging #894
> <https://codecov.io/gh/profclems/glab/pull/894?src=pr&el=desc&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Clement+Sam>
> (a684bfa
> <https://github.com/profclems/glab/commit/a684bfa53a1da46166705ef848e833e0d9641e2d>)
> into docs-markdown
> <https://codecov.io/gh/profclems/glab/commit/88df561ca57b6f7f9b45f28cfffed6d95c4dcb3c?el=desc&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Clement+Sam>
> (88df561
> <https://github.com/profclems/glab/commit/88df561ca57b6f7f9b45f28cfffed6d95c4dcb3c>)
> will *not change* coverage.
> The diff coverage is n/a.
>
> [image: Impacted file tree graph]
> <https://codecov.io/gh/profclems/glab/pull/894?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Clement+Sam>
>
> @@              Coverage Diff               @@
> ##           docs-markdown     #894   +/-   ##
>
> ==============================================
>
>   Coverage          59.41%   59.41%
>
> ==============================================
>
>   Files                 93       93
>
>   Lines               7341     7341
>
> ==============================================
>
>   Hits                4362     4362
>
>   Misses              2599     2599
>
>   Partials             380      380
>
> ------------------------------
>
> Continue to review full report at Codecov
> <https://codecov.io/gh/profclems/glab/pull/894?src=pr&el=continue&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Clement+Sam>
> .
>
> *Legend* - Click here to learn more
> <https://docs.codecov.io/docs/codecov-delta?utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Clement+Sam>
> Δ = absolute <relative> (impact), ø = not affected, ? = missing data
> Powered by Codecov
> <https://codecov.io/gh/profclems/glab/pull/894?src=pr&el=footer&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Clement+Sam>.
> Last update 88df561...a684bfa
> <https://codecov.io/gh/profclems/glab/pull/894?src=pr&el=lastupdated&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Clement+Sam>.
> Read the comment docs
> <https://docs.codecov.io/docs/pull-request-comments?utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Clement+Sam>
> .
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/profclems/glab/pull/894#issuecomment-961062462>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AMWPMFZDVHUY5WI7WKVKOA3UKKKSPANCNFSM5HGH4DMA>
> .
>

Spreadcat committed:
fix: add linebreak to check-output msg (#895)

This fix adds a linebreak after the check-update output message.

Co-authored-by: Daniel Buøy-Vehn <profclem.gitlab@micronarrativ.org>

kartikay101 committed:
fix(mr_create/issue_create): add encoding to title and change description encoding (#871)

* fix(mr_create/issue_create): add encoding to title and change description encoding

- Prevent browsers from converting a space containing query to a search query
- Allow users to add symbols to their title and description
    - QueryEscape does not escape '+' which is converted to a space by the browser

* fix(issue_create) : remove unused repo parameter

* fix(issue_create/mr_create) : manually encode + to %2B

* fix(issue_create/mr_create) : prevent open from auto encoding characters

The tick on the URL makes `open` on a mac encode it, so already encoded data such as a %20 is encoded to %2520
Removing the utf8=`tick` prevents auto encoding and makes urls work as is allowing symbols to be used in title and description

Co-authored-by: Kartikay Shandil <kartikays@sahaj.ai>

carlpett committed:
feat(mr): Only output MR URL to stdout if output is not a TTY (#883)

* feat(mr): Only output MR URL to stdout if output is not a TTY

This enables "glab mr create -f -y | some command" to only pass the url
through the pipe.

* feat(issue): Output only issue URL to stdout if output is not a TTY

bradym committed:
feature: add variable update command (#902)

kampka committed:
feature: Read config directory from GLAB_CONFIG_DIR (#921)

When working with many different gitlab instances for different
projects, the global config quickly gets cluttered up with
specific settings that do not clean up conveniently. On the other hand,
local configs are to local, restricting the config to one repository
instead of one project or instance that potentially contains many
relevant repositories.
This change introduces a middle ground solution via the GLAB_CONFIG_DIR
environment variable. Setting this variable will allow global
configuration to travel with the project instead of populating the XDG
global config.

NikhilMJagtap committed:
added push mirror and command (#887)

* added push mirror and command

* reformatted push code to separate from pull mirror

* restructured mirror calls

* fixed typo in long description, changed maxArgs to exactArgs, added edit call for pull mirror

* made url and direction required, passing allow-divergence now warns when used with pull mirror, added separate options struct for both push and pull mirror calls

* add prject url to be specified as argument

Co-authored-by: NikhilMJagtap <nikhil.jagtap@workindia.in>
Co-authored-by: Clement Sam <clementsam75@gmail.com>

zemzale committed:
fix: Write aliases to symlink if it exists (#930)

In cases where `aliases.yaml` was smylinked to some other place(dotfiles
for example), it would remove the smylink and write to a new file.

This commit fixes that by resolving the smylink before writing to the
file.

Issue #892

BreD1810 committed:
chore: Update Arch README link to official repos

zemzale committed:
Merge pull request #937 from BreD1810/arch-readme-update

chore: Update Arch README link to official repos

zemzale committed:
feat: Add group flag to mr list command (#931)

* feat: Add group flag to mr list command

This adds a flag for listing groups mrs. This works only for groups in
authenticated repos. This is acceptable since mr list only works on
authenticated repos too.

Issue #909

* feat: Add group flag for issue listing

Issue #909

invalid-email-address committed:
docs(update): v1.22.0

alerque committed:
chore: Correct Arch Linux install command

* Use official package manager (pacman) instead of unsupported AUR
  helper / pacman wrapper (yay) that was only necessary for unofficial
  packaging.

* Drop db sync flag (-y) because doing that while installing packages
  results in an unsupported partial-update scenario. The correct usage
  is to update the whole system first (if necessary) with `pacman Syu`,
  but Arch users will know that, so just the single relevant install
  command is best for projects to show.

zemzale committed:
Merge pull request #938 from alerque/arch-pacman

chore: Correct Arch Linux install command

justinTM committed:
accept GITLAB_ACCESS_TOKEN (#940)

zemzale committed:
feat(repo/clone): Add options for page and page size (#942)

This adds the missing options for setting page number and page size for
repo clone to allow for cloning more then 100 repos.

Issue #941

LitoMore committed:
Fix hyperlinks rendering in less (#936)

* Fix hyperlinks rendering in less

* Fix multi-append

* Fix logic of less env

jpeedroza committed:
Add artifact download command (#910)

* feat: add artifact download command

* fix(command/ci/artifact): add error return in functions

* fix: add error return in copy function

profclems committed:
goreleaser: witch to `use` instead of `use_buildx` (#944)

use_buildx is deprecated in favor of the more generalist use

https://goreleaser.com/deprecations/\#dockeruse_buildx

cfebs committed:
feat: add `repo list` command (#906)

* Add repo list

* Fix title

* Add filters, add ssh urls

* linting

Co-authored-by: Brendan O'Leary <boleary@gitlab.com>
Co-authored-by: Collin Lefeber <lefeberc@gmail.com>
Co-authored-by: Clement Sam <clementsam75@gmail.com>

matrixise committed:
fix: Example of glab ci list --status (#946)

x3ro committed:
feat: add transfer command (#929)

* Add command for transferring repositories

* minor fixes

Co-authored-by: Clement Sam <clementsam75@gmail.com>

zemzale committed:
fix: Don't show non-templates when asking chosing (#927)

* feat(git): Allow for overriding the TopLevelDir

This would allow us to use a different output for this command in tests.

Issue #923

* fix: Don't show non-templates when asking chosing

In cases where users have non template files in the .gitlab/[template]/
folder they might will be picked up and displayed.

This fixes that by filtering out all the files that are hidden and don't
end in .md

Issue #923

* feat(cmdutils): Sort templates in alphabetical order

zemzale committed:
feat(api): Add issue time wrapper function

These are building blocks for adding time tracking functionality for
issues.

Issue #934

zemzale committed:
Merge pull request #947 from zemzale/feat-time-api

feat(api): Add issue time wrapper function

zemzale committed:
feat(issue): Add time estimate setting when creating

This moves all the post creation actions into a separate function. This
now includes adding estimates for the issue.

This will be followed up with adding time spent too.

Issue #934

zemzale committed:
feat(issue): Add time spent to create

This adds two more flags for issue create that allow to setup time
estimates and time spent values.

Issue #934

zemzale committed:
feat(api): Add api wrappers for creating snippets

This adds two simple API wrappers for creating snippets for the user and
inside the project. This should help us start work with creating a
snippets command.

Issue #872

byteknacker committed:
Change wrong flag

zemzale committed:
feat(snippet): Add command for creating snippets

This adds a simple snippet creation command. This currently just allows
for creating snippets from STDIN like so:

`echo "this is a script" | glab snippet create --title "snippet" --filename "example.py"`

There are few things that are missing from this, like ability to add
more then one file to the snippet, but that is not supported by the "github.com/xanzy/go-gitlab" yet.

Issue is opened for that https://github.com/xanzy/go-gitlab/issues/1372

Issue #872

zemzale committed:
Merge pull request #950 from zemzale/feat-snippets-api

feat(api): Add api wrappers for creating snippets

byteknacker committed:
Make flag verbose for -o of grep command

zemzale committed:
Merge pull request #951 from byteknacker/trunk

Change flag which causes errors in some shells

hbre committed:
Specified Full Package name for winget Installation (#968)

WinGet Install needs full qualified package name:

carlpett committed:
fix: Fail with a specific error when unauthenticated (#886)

* fix: Fail with a specific error when unauthenticated

If resolutions fails, return the error instead of "no GitLab Projects found
from remotes"

* fix: Do not break multi-network setups on authentication failure

* go mod tidy

a333klm committed:
fix(authentication): hostnamevalidator accepts folders (#961)

* fix(authentication): hostnamevalidator accepts folders

- the hostname validator did not accept hostnames like localhost:8888/gitlab

Fixes #960

* fix test

Co-authored-by: Clement Sam <clementsam75@gmail.com>

zemzale committed:
Merge pull request #949 from zemzale/feat-time-issue-create

feat(issue): Add time tracking functionality

justin-f-perez committed:
docs: fix copypasta errors for `glab issue view`

* Fix copypasta in CLI `glab issue view --help`

lpapp-foundry committed:
feat: provide /reviewer quick action

This is when creating a merge request using the command line option
git mr create --reviewer -w. Other options, like -a and -l, have been
working, but this seems to have gone missed.

Delivers #974

zemzale committed:
Merge pull request #975 from lpapp/issue-974

feat: provide /reviewer quick action

lpapp-foundry committed:
docs: change --labels to --label

The correct parameter seems to be --label (singular) as opposed to
--labels (plural).

It is a bit strange that the same example is given in two different
places, so this may need centralisation in one place in the future.

ygarasab committed:
fix: fix validation for GitLab 14.5 tokens (#971)

zemzale committed:
Merge pull request #976 from lpapp/doc-fix-labels

docs: change --labels to --label

zemzale committed:
Merge pull request #973 from justin-f-perez/patch-1

fix: copypasta errors for `glab issue view --help`

zemzale committed:
Merge pull request #952 from zemzale/feat-snippet-cmd

feat(snippet): Add command for creating snippets

meleu committed:
fix broken link to "quick install"

meleu committed:
leftover

zemzale committed:
Merge pull request #981 from meleu/patch-1

fix broken link to "quick install"

meleu committed:
'pipeline ci' is deprecated (#996)

https://github.com/profclems/glab/issues/372

profclems committed:
announce adoption by GitLab in the readme (#1017)

See #983

profclems committed:
Update README.md

profclems committed:
Delete gitlab-issues-to-github.yml

braswelljr committed:
Rebase and Merge Refs for Docs generation
